crmd/remote_lrmd_ra.c

```diff
@@ -80,6 +80,10 @@ typedef struct remote_ra_data_s {
     enum remote_migration_status migrate_status;
 
     gboolean active;
+    gboolean is_maintenance; /* kind of complex to determine from crmd-context
+                              * so we have it signalled back with the
+                              * transition from pengine
+                              */
```

Comment:
adding the handling of this operation to crmd as well of course does the trick but I'm
slightly loosing confidence in if the pseudo-action is the correct tool for the task.